importService/importProductsFile.js

```diff
@@ -0,0 +1,17 @@
+const AWS = require('aws-sdk');
```

Comment:
It's better to use aws-sdk v3 - better promise support, typings, small package size